### PR TITLE
[textanalytics] add :keyword: to docstrings for TextDocumentInput and DetectLanguageInput

### PR DESCRIPTION
**Pull request by a-noyass**
https://github.com/Azure/azure-sdk-for-python/issues/16482

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py [hunk 1]
```diff
@@ -740,6 +740,12 @@ def __repr__(self):
 class DetectLanguageInput(LanguageInput):
     """The input document to be analyzed for detecting language.
 
+    :keyword str id: Unique, non-empty document identifier.
```
Comment (iscai-msft) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:743]: hey @a-noyass thank you so much for doing this PR.

When adding the keyword docstrings, you can remove the `ivar` ones, so they're not duplicated. Other than that your pr looks great!

**Review by iscai-msft — APPROVED** [review]

**Review by kristapratico — APPROVED** [review]
Looks good :)

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py (hunk 1 above)
Comment (iscai-msft) [reply sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:743]: actually @a-noyass i'm totally wrong, we should keep both. Going to merge your PR when it's green

Comment (a-noyass) [reply sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py:743]: @iscai-msft Okay. Thank you!

**Pull request merged**